tcl/pd_bindings.tcl

```diff
@@ -87,7 +87,9 @@ proc ::pd_bindings::global_bindings {} {
     bind all <$::modifier-Shift-Key-I> {menu_send %W hradio}
     bind all <$::modifier-Shift-Key-L> {menu_clear_console}
     bind all <$::modifier-Shift-Key-M> {menu_message_dialog}
+    bind all <quotedbl> {::entry_canvasmsg::openclose_canvasmsg $::focused_window}
```

Comment:
note that the `"` (double-quote) is found on very different places on various keyboards.
On my German keyboard, it's definitely **not** on <kbd>Shift</kbd>+<kbd>'</kbd>